src/screens/Dashboard/ActivityPanel/index.js

```diff
@@ -29,7 +29,7 @@ export default class ActivityPanel extends Component {
           <FlatList
             renderItem={(item) => {
               const message = item.item;
-              const messageText = message.interpret ? i18n.t(message.msg) : message.msg;
+              const messageText = message.interpret ? i18n.t('activity_log.'+message.msg, JSON.parse(message.params)) : message.msg;
```

Comment:
Please change activity_log to camel case. Consider using string interpolation with backticks string (\`\`) instead of concatenation - it's a [part](https://github.com/airbnb/javascript#es6-template-literals) of Airbnb style we start using, however discussion about style is opened.